tests/unit/s2n_3des_test.c

```diff
@@ -58,18 +58,17 @@ int main(int argc, char **argv)
     EXPECT_SUCCESS(s2n_hmac_init(&conn->secure.server_record_mac, S2N_HMAC_SHA1, mac_key, sizeof(mac_key)));
     conn->actual_protocol_version = S2N_TLS11;
 
-    int max_aligned_fragment = S2N_DEFAULT_FRAGMENT_LENGTH - (S2N_DEFAULT_FRAGMENT_LENGTH % 8);
-    for (int i = 0; i <= max_aligned_fragment + 1; i++) {
+    for (int i = 0; i <= S2N_DEFAULT_FRAGMENT_LENGTH + 1; i++) {
         struct s2n_blob in = {.data = random_data,.size = i };
         int bytes_written;
 
         EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->out));
         EXPECT_SUCCESS(bytes_written = s2n_record_write(conn, TLS_APPLICATION_DATA, &in));
 
-        if (i < max_aligned_fragment - 20 - 8 - 1) {
+        if (i <= S2N_DEFAULT_FRAGMENT_LENGTH) {
             EXPECT_EQUAL(bytes_written, i);
```

Comment:
Would it be the same as:

```
EXPECT_EQUAL(bytes_written, MIN(i, S2N_DEFAULT_FRAGMENT_LENGTH));
```

Not sure if that makes it easier to understand. But it seems like the branch exists mainly for the `+1` case?

Reply:
> it seems like the branch exists mainly for the +1 case?

right. I would prefer the if statement. the branching makes it clear in which conditions bytes_written gets bounded.